EnglishFilter/sections/antiadblock.txt

```diff
@@ -550,6 +550,9 @@ kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/75599
+imginn.com#%#//scriptlet("abort-current-inline-script", "document.createElement", "unblocker")
+imginn.com#@#.ad_slot
```

Comment:
`||copyrightcontent.org/unblocker/$domain=imginn.com` is ok. Also we have a similar rule, so just add a domain.
As I said earlier. don't add rules at the beginning, just a little lower.
For example
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/108958877-a7ecb280-7684-11eb-920b-83270eb6a15a.png)

</details><br/>

Reply:
@zloyden What is wrong If I use scriptlet?

Reply:
Everything works perfectly even for content blockers.

Reply:
It's not necessary here. The best option is considered to be a simpler solution.